src/Controls/src/Core/Handlers/Items/Android/Adapters/ReorderableItemsViewAdapter.cs

```diff
@@ -10,9 +10,12 @@ public class ReorderableItemsViewAdapter<TItemsView, TItemsViewSource> : Groupab
 		where TItemsView : ReorderableItemsView
 		where TItemsViewSource : IGroupableItemsViewSource
 	{
+		private readonly bool _hasHeader;
+
 		public ReorderableItemsViewAdapter(TItemsView reorderableItemsView,
 			Func<View, Context, ItemContentView> createView = null) : base(reorderableItemsView, createView)
 		{
+			_hasHeader = reorderableItemsView.Header is not null;
```

Comment:
Can 't this be null and but we have a HeaderTemplate?

Reply:
If HeaderTemplate is present, this value is not null. I tested it with HeaderTemplate in my examples.